feat: FrameworkTemplatePool rework

ebariche opened the pull request:
## PR Type

What kind of change does this PR introduce?
- Feature

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

ebariche committed:
chore: Revert force disable template pooling for all platforms

ebariche committed:
feat: FrameworkTemplatePool rework

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

jeromelaban commented on src/Uno.UI.Tests/Windows_UI_XAML_Controls/ItemsControlTests/Given_ItemsControl.cs:
```diff
@@ -524,7 +524,7 @@ public void When_Collection_Reset()
 			}
 
 			Assert.AreEqual(SUT.Items.Count, 5);
-			Assert.AreEqual(count, FrameworkTemplatePool.IsPoolingEnabled ? 5 : 8);
```
Let's keep those, just in case.

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Explicitly release template to the pool

jeromelaban committed:
chore: Restore non-pooled asserts

jeromelaban committed:
chore: Restore template pool android validation

jeromelaban committed:
chore: Restore frameworktemplatepool test

jeromelaban committed:
chore: Recycle root on template/datatemplated changed

jeromelaban committed:
chore: Adjust partial class

jeromelaban committed:
chore: Restore behavior for https://github.com/unoplatform/uno/pull/8041

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Adjust release order

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Adjust test generation

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Recycle template on content changing

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Adjust ContentPresenter template recycling

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Don't track instances when pooling is disabled

jeromelaban committed:
chore: Cleanup template lists when empty

jeromelaban committed:
chore: Remove instances when tracking is cancelled

jeromelaban committed:
chore: Scavenge pool on GC assertion

jeromelaban committed:
chore: Avoid leak from template closures

jeromelaban committed:
chore: Clear pool on manual template activation

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Always recycle explicitly dequeued template instances

jeromelaban committed:
chore: Fix incorrect root inclusion

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Validate that instance trackers are removed

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Adjust for missing using

jeromelaban committed:
chore: Adjuste comments, method names

jeromelaban committed:
chore: Validate for tracker references cleanup

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Fix wasdk build

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

nventive-devops commented:
The [build 112369](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=112369) found UI Test snapshots differences: `android-28-net7`: **0**, `android-28-net7-Snap`: **44**, `ios`: **110**, `ios-Snap`: **298**, `skia-gtk-screenshots`: **17**, `skia-wpf-screenshots`: **16**, `wasm`: **69**, `wasm-automated-net7.0-UWP-Default-automated`: **38**, `wasm-automated-net7.0-UWP-RuntimeTests-0`: **0**, `wasm-automated-net7.0-UWP-RuntimeTests-1`: **0**, `wasm-automated-net7.0-UWP-RuntimeTests-2`: **0**, `wasm-automated-net7.0-WinUI-Benchmarks-automated`: **0**, `wasm-automated-net7.0-WinUI-RuntimeTests-0`: **0**, `wasm-automated-net7.0-WinUI-RuntimeTests-1`: **0**, `wasm-automated-net7.0-WinUI-RuntimeTests-2`: **0**

<details>
<summary>Details</summary>


* `android-28-net7`: **0** changed over 1

* `android-28-net7-Snap`: **44** changed over 1051

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ListView_ListViewIncrementalLoading_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewIncrementalLoading`
  - `NavigationView_MUXControlsTestApp_NavigationViewRS4Page_MUXControlsTestApp_NavigationViewRS4Page`
  - `BreadcrumbBar_MUXControlsTestApp_BreadcrumbBarPage_MUXControlsTestApp_BreadcrumbBarPage`
  - `MediaPlayerElement_MediaPlayerElement_Stretch_Fill_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Stretch_Fill`
  - `MediaPlayerElement_Mini_player_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Minimal`
  - `ListView_ListViewLargeLegacy_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewLargeLegacy`
  - `NavigationView_MUXControlsTestApp_NavigationViewTopNavPage_MUXControlsTestApp_NavigationViewTopNavPage`
  - `Default_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ListView_ListView_IsSelected_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ListView_ListView_IsSelected`
  - `Gesture_Recognizer_Pointer_Events_test_bench_UITests_Shared_Windows_UI_Input_GestureRecognizer_PointersEvents`
  - `MediaPlayerElement_Using_3gp_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_3gp_Extension`
  - `MediaPlayerElement_Using_flv_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Flv_Extension`
  - `TabView_UITests_Microsoft_UI_Xaml_Controls_TabViewTests_TabViewPage_UITests_Microsoft_UI_Xaml_Controls_TabViewTests_TabViewPage`
  - `Expander_WinUIExpanderPage_UITests_Shared_Microsoft_UI_Xaml_Controls_ExpanderTests_WinUIExpanderPage`
  - `Icons_UITests_Microsoft_UI_Xaml_Controls_ImageIconTests_ImageIconPage_UITests_Microsoft_UI_Xaml_Controls_ImageIconTests_ImageIconPage`
  - `ListView_ListViewGrouped_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewGrouped`
  - `MediaPlayerElement_Using_mkv_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Mkv_Extension`
  - `MediaPlayerElement_Using_mov_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Mov_Extension`
  - `MediaPlayerElement_Using_mp3_Audio_only_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Mp3_Extension`
  - `MediaPlayerElement_Using_ogg_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Ogg_Extension`
  - `Progress_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_ProgressRingHeightTest_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_ProgressRingHeightTest`
  </details>

* `ios`: **110** changed over 266

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Check_CornerRadius_Border_UITests_Shared_Windows_UI_Xaml_Controls_BorderTests_Border_CornerRadius_Toggle`
  - `ComboBoxTests_ToggleDisabled_ComboBox_Disabled`
  - `ListView_ExpandableItemLarge_ExpandHeaderWithMultipleItems_Validation_Initial_State`
  - `ListView_ExpandableItemLarge_ExpandHeaderWithMultipleItems_Validation_Expanded_State`
  - `ListView_ExpandableItem_ExpandSingleItem_Collapsed_State`
  - `ListView_ExpandableItem_ExpandMultipleItems_Initial_State`
  - `ListView_SelectedItem_Both_Selection_Changed`
  - `ListView_ObservableCollection_Creation_Count_UITests_Windows_UI_Xaml_Controls_ListView_ListView_ObservableCollection_CreationCount`
  - `Pivot_Non_PivotItem_Items_UITests_Shared_Windows_UI_Xaml_Controls_Pivot_Pivot_CustomContent_Automated`
  - `ProgressRing_Visibility_Collapsed_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `When_Background_Updated_After_SolidColorBrush_Color_update`
  - `When_CustomContent_CommandBarTitleShouldBeVisible_NativeFrame_UITests_Windows_UI_Xaml_Controls_CommandBar_CustomContent_CommandBar_Frame`
  - `When_ExtendedSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `Check_CornerRadius_Border_No_CornerRadius`
  - `Keyboard_Textbox_NoScrollViewer_Validation_1_-_Remove_Focus_on_filledTextBox`
  - `ListView_ExpandableItemLarge_ExpandHeaderWithSingleItem_Validation_Collapsed_State`
  - `ListView_ExpandableItem_ExpandSingleItem_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_Expandable_Item`
  - `ListView_Weird_Measure_During_Arrange_after_layout`
  - `ScrollViewer_Removed_And_Added_UITests_Windows_UI_Xaml_Controls_ScrollViewerTests_ScrollViewer_Add_Remove`
  - `Check_ListView_Swallows_Measure_before_scroll`
  </details>

* `ios-Snap`: **298** changed over 1042

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Border_Border_CornerRadius_Gradient_UITests_Windows_UI_Xaml_Controls_BorderTests_Border_CornerRadius_Gradient`
  - `Border_OnepxThicknessandRadius_Uno_UI_Samples_UITests_BorderTestsControl_OnepxThicknessandRadius`
  - `Brushes_EllipsemaskingEllipseGrid_Uno_UI_Samples_UITests_ImageBrushTestControl_EllipsemaskingEllipseGrid`
  - `Brushes_EllipsemaskingEllipseSuperpositionPanel_Uno_UI_Samples_UITests_ImageBrushTestControl_EllipsemaskingEllipseSuperpositionPanel`
  - `Buttons_Custom_Button_With_ContentTemplate_Uno_UI_Samples_Content_UITests_ButtonTestsControl_Custom_Button_With_ContentTemplate`
  - `Clipping_SamplesApp_Windows_UI_Xaml_Clipping_ButtonClippingTestsControl_SamplesApp_Windows_UI_Xaml_Clipping_ButtonClippingTestsControl`
  - `ComboBox_ComboBox_Picker_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Picker`
  - `ContentControl_ContentControl_DefaultText_Uno_UI_Samples_Content_UITests_ContentControlTestsControl_ContentControl_DefaultText`
  - `Dialogs_ContentDialog_Simple_UITests_Shared_Windows_UI_Xaml_Controls_ContentDialogTests_ContentDialog_Simple`
  - `Flyouts_UITests_Shared_Windows_UI_Xaml_Controls_MenuFlyoutTests_UIElement_ContextFlyout_UITests_Shared_Windows_UI_Xaml_Controls_MenuFlyoutTests_UIElement_ContextFlyout`
  - `Flyouts_UITests_Windows_UI_Xaml_Controls_FlyoutTests_Flyout_ShowAt_Window_Content_UITests_Windows_UI_Xaml_Controls_FlyoutTests_Flyout_ShowAt_Window_Content`
  - `Flyouts_UITests_Windows_UI_Xaml_Controls_Flyout_Flyout_TemplatedParent_UITests_Windows_UI_Xaml_Controls_Flyout_Flyout_TemplatedParent`
  - `Flyouts_UITests_Windows_UI_Xaml_Controls_MenuFlyoutTests_MenuFlyoutSubItem_Placement_UITests_Windows_UI_Xaml_Controls_MenuFlyoutTests_MenuFlyoutSubItem_Placement`
  - `Image_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Vertical_UITests_Shared_Windows_UI_Xaml_Controls_ImageTests_Image_Stretch_Algmnt_Inf_Vertical`
  - `ListView_ListViewLarge_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewLarge`
  - `ListView_ListViewLegacy_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewLegacy`
  - `ListView_ListViewMarginTest_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewMarginTest`
  - `ListView_ListView_ItemClick_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_ItemClick`
  - `ListView_ListView_ItemContainerStyleSelector_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_ItemContainerStyleSelector`
  - `ListView_ListView_WithScrollViewer_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_WithScrollViewer`
  </details>

* `skia-gtk-screenshots`: **17** changed over 1100

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Microsoft.UI.Composition-CompositionVisualSurface.png`
  - `Image-Image_Formats.png`
  - `Pickers-CalendarView_Theming.png`
  - `Microsoft.UI.Composition-CompositionMaskBrush.png`
  - `Microsoft.UI.Composition-RedirectVisual.png`
  - `ListView-ListViewSelectedItems.png`
  - `MUX-WinUIProgressRingPage.png`
  - `Progress-WinUIProgressRing_Features.png`
  - `Image-Image_Stretch_None.png`
  - `Progress-WinUIProgressRingPage.png`
  - `Progress-WinUIProgressRing_CustomSources.png`
  - `Shapes-Offscreen_Shapes.png`
  - `PipsPager-PipsPagerExamples.png`
  - `Progress-WindowsProgressRing_GH1220.png`
  - `MUX-WinUIProgressRing_CustomSources.png`
  - `Progress-ProgressRingHeightTest.png`
  - `WinUI-PipsPagerExamples.png`
  </details>

* `skia-wpf-screenshots`: **16** changed over 1100

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Pickers-CalendarView_Theming.png`
  - `ListView-ListViewSelectedItems.png`
  - `Microsoft.UI.Composition-CompositionVisualSurface.png`
  - `Microsoft.UI.Composition-RedirectVisual.png`
  - `MUX-WinUIProgressRingPage.png`
  - `MUX-WinUIProgressRing_CustomSources.png`
  - `Progress-WinUIProgressRingPage.png`
  - `Progress-ProgressRingHeightTest.png`
  - `Progress-WindowsProgressRing_GH1220.png`
  - `MediaPlayerElement-Mini player.png`
  - `Progress-WinUIProgressRing_CustomSources.png`
  - `Progress-WinUIProgressRing_Features.png`
  - `Microsoft.UI.Composition-CompositionMaskBrush.png`
  - `Automation-AutomationProperties_Name.png`
  - `PipsPager-PipsPagerExamples.png`
  - `WinUI-PipsPagerExamples.png`
  </details>

* `wasm`: **69** changed over 1047

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListViewLegacy`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ComboBox.ComboBox_DropDownPlacement`
  - `Uno.UI.Samples.Content.UITests.CommandBar.CommandBar_Examples`
  - `MUXControlsTestApp.PaneLayoutTestPage`
  - `UITests.Shared.Windows_UI_Xaml_Controls.NavigationViewTests.NavigationView_TopNavigation`
  - `UITests.Windows_UI_ViewManagement.UISettingsTests`
  - `UITests.Windows_UI_Xaml_Controls.ProgressRing.WindowsProgressRing_GH1220`
  - `Benchmarks.Shared.Controls.BenchmarkDotNetTestsPage`
  - `UITests.Microsoft_UI_Xaml_Controls.ProgressRing.WinUIProgressRingPage`
  - `UITests.Microsoft_UI_Xaml_Controls.ProgressRing.WinUIProgressRing_CustomSources`
  - `UITests.Windows_UI_Xaml_Controls.TextBlockControl.TextBlock_TextTrimming`
  - `MUXControlsTestApp.NavigationViewCompactPaneLengthTestPage`
  - `MUXControlsTestApp.NavigationViewMenuItemStretchPage`
  - `MUXControlsTestApp.NavigationViewStretchPage`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Padding`
  - `UITests.Microsoft_UI_Xaml_Controls.NumberBoxTests.NumberBox_Header`
  - `UITests.Microsoft_UI_Xaml_Controls.RatingControlTests.RatingControlPage`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ListView.ListView_ItemsPanel_HotSwap`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ListView.ListView_Aligned_Left`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ListView.ListView_With_ListViews_Count_Measure`
  </details>

* `wasm-automated-net7.0-UWP-Default-automated`: **38** changed over 1952

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Default_StrokeThickness_MyLine_0_StrokeThickness`
  - `ListView_ListView_Expandable_Item_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_Expandable_Item`
  - `MinMaxTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `Default_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ListView_ListView_IsSelected_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ListView_ListView_IsSelected`
  - `MediaPlayerElement_Using_mp3_Audio_only_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Mp3_Extension`
  - `MediaPlayerElement_Using_ogg_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Ogg_Extension`
  - `ListView_HorizontalListViewGrouped_SamplesApp_Windows_UI_Xaml_Controls_ListView_HorizontalListViewGrouped`
  - `MediaPlayerElement_MediaPlayerElement_Stretch_Fill_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Stretch_Fill`
  - `MediaPlayerElement_Mini_player_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Minimal`
  - `MediaPlayerElement_Sources_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Sources`
  - `MediaPlayerElement_Using_3gp_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_3gp_Extension`
  - `ColorPicker_WinUIColorPickerPage_UITests_Shared_Microsoft_UI_Xaml_Controls_ColorPickerTests_WinUIColorPickerPage`
  - `Progress_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIProgressRing_Features_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIProgressRing_Features`
  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `Path_UITests_Shared_Windows_UI_Xaml_Shapes_PathTestsControl_SVG_Path_UITests_Shared_Windows_UI_Xaml_Shapes_PathTestsControl_SVG_Path`
  - `ListView_ListViewSelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `TextBlock_TextBlockSimpleContrainedHorizontalCenterWrap2_Uno_UI_Samples_Content_UITests_TextBlockControl_TextBlockSimpleContrainedHorizontalCenterWrap2`
  - `TextBlock_Textblocktimespancustomformat_Uno_UI_Samples_Content_UITests_TextBlockControl_Textblocktimespancustomformat`
  - `Pickers_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming_UITests_Windows_UI_Xaml_Controls_CalendarView_CalendarView_Theming`
  - `PipsPager_MUXControlsTestApp_PipsPagerExamples_MUXControlsTestApp_PipsPagerExamples`
  </details>

* `wasm-automated-net7.0-UWP-RuntimeTests-0`: **0** changed over 1

* `wasm-automated-net7.0-UWP-RuntimeTests-1`: **0** changed over 1

* `wasm-automated-net7.0-UWP-RuntimeTests-2`: **0** changed over 1

* `wasm-automated-net7.0-WinUI-Benchmarks-automated`: **0** changed over 1

* `wasm-automated-net7.0-WinUI-RuntimeTests-0`: **0** changed over 1

* `wasm-automated-net7.0-WinUI-RuntimeTests-1`: **0** changed over 1

* `wasm-automated-net7.0-WinUI-RuntimeTests-2`: **0** changed over 1

</details>

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
chore: Move cookie tracker to use weak references to reduce staged collection

jeromelaban committed:
chore: Add support for removing strong parent changed callbacks

jeromelaban committed:
chore: Unregister parent changed callback on parent set to null

This avoids possible memory leak caused by a strong reference to the pool though parent change notifications

jeromelaban committed:
chore: Force scoped templatepool cleanup in tests if pool is disabled

jeromelaban committed:
chore: Add leak testing when pooling is enabled

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html

jeromelaban committed:
fix: Infobar memory leak

jeromelaban committed:
fix: Make Binding.Source weak for UIElement instances.

jeromelaban committed:
fix: Memory leak in ScrollViewer registration to child size changed

jeromelaban committed:
fix: Adjust Load/Unload reference to parent in VisualStateGroup

jeromelaban committed:
chore: Fix recycled instance skipping

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-15336/index.html

unodevops commented:
🤖 Your WebAssembly Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-15336/index.html
